Dockerfile

```diff
@@ -35,3 +35,5 @@ RUN apt-get update && \
 RUN ln -s /usr/nginx/sbin/nginx /usr/sbin/nginx
 
 COPY . /lua-resty-jwt/
+
+RUN ls /lua-resty-jwt/testcerts
```

Comment:
hmm... you want to make sure the directory exists?

that's okay. i think i'll write a better dockerfile.
this one is too s....l.....o....w.....